fsck/repair.c

```diff
@@ -61,6 +61,7 @@ static struct exfat_repair_problem problems[] = {
 	{ER_FILE_LARGER_SIZE, ERF_PREEN_YES, ERP_TRUNCATE, 0, 0, 0},
 	{ER_FILE_DUPLICATED_CLUS, ERF_PREEN_YES, ERP_TRUNCATE, 0, 0, 0},
 	{ER_FILE_ZERO_NOFAT, ERF_PREEN_YES, ERP_FIX, 0, 0, 0},
+	{ER_DIR_FIRST_CLUS, ERF_PREEN_YES, ERP_TRUNCATE, 0, 0, 0}
```

Comment:
I think ERP_FIX is better than ERP_TRUNCATE. because this makes user's asking "FIx?".